### ISSUE TITLE
Get feedback from informal user testing

### ISSUE THREAD

**Issue by philectron** [osu-capstone-cs72/cs-applied-plan-portal#67]
We should ask people to navigate through our application and get their feedback.

**Comment by philectron** [comment]
Put feedback in the comment section.

**Comment by silverware13** [comment]
Don't think we really have time to implement any changes based on user testing beyond the class reviews we already had. I am going to defer this issue

**Comment by philectron** [comment]
Yeah, I managed to get 1 person (maybe 2 people) to test our app 😅 . This is truly a difficult time to contact people. We could have moved along in this task way faster if we had been able to just meet people and show them a local version of the app.

That being said, feel free to put any feedback to the comment section because it may be useful for the next team if this project is continued after this year's capstone.

**Comment by philectron** [comment]
Note: Each comment is the feedback from a distinct person.

**Comment by philectron** [comment]
This person was impressed at the UX. She was able to log in and replicate her applied plan. She was able to fill in the courses and submit the plan.

**Comment by philectron** [comment]
This person said that the course search bar could be more flexible: the search text `CS493` should be accepted, and the system should show the same search results as if the user enters `CS 493`.

Anecdotally, I usually find myself lazy and type `CS493` very frequently in OSU class search.

**Comment by philectron** [comment]
This showed a validation error when this person tried to break it, which is good:

![image](https://user-images.githubusercontent.com/14208546/82113638-6ea61000-970c-11ea-85b1-c8e751646abd.png)

![image](https://user-images.githubusercontent.com/14208546/82113672-9ac19100-970c-11ea-8cde-f50bb91ace55.png)

However, he said "I think your error handling needs to draw more attention to itself, especially because there are a few different places where errors show up maybe with a popup, even."

He thinks the Notification button on the navbar should be click-open, not hover-open. For example, GitHub and Twitter (even though these two examples link to a new page).

He liked everything else!

**Comment by silverware13** [comment]
This user is not a Computer Science major. She tried the application but didn't know too much about applied plans.

She was able to figure out how the application was meant to work very quickly.
She tried adding a course and saw that it rejected her request and asked "Why can't I take this course?" then she looked back at the screen a second later and realized that there was an error message, she then realized due to the error message that she couldn't take graduate courses.

She had no idea how many courses she needed to take. Around 15 credits she tried to submit her plan. At this point she was told that she needed at least 32 credits. She added more courses until she had the credits and submitted.

She looked over her plan. As she did an advisor started reviewing her plan, she left comments in response. When she was done she went to the home page and saw her one course. She mentioned that the page looked very empty with just a single tiny course in middle of empty space.
